### DISCUSSION TITLE
How do I change the formatting rules?

### DISCUSSION THREAD

**Discussion by HarrisonHemstreet** [LunarVim/LunarVim#2789] [Q&A] [answered]
I'm currently using LunarVim while editing some React files. We are using 4 spaces in this repo. However, LunarVim is changing my 4 spaces into two on save... How do I prevent this from happening?

I've looked in both `lv-settings.lu` and my `config.lua`. I have not been able to find the line in either files that controls the indentation setting.

**Comment by LostInTheLogs** [comment, marked as answer]
check the formatter settings and `:h shiftwidth`
